### ISSUE TITLE
Release LemMinX 0.11.0

### ISSUE THREAD

**Issue by fbricon** [eclipse/lemminx#619]
- [ ] make sure jars are properly signed
- [x] automate release process + git tagging
- [x] release Maven jars to repo.eclipse.org (Bonus points for Maven Central release)

**Comment by mickaelistria** [comment]
We're really in need of a 1st release to make substantial progress with Wild Web Developer and m2e adopting Lemminx, so it'd be great if this can happen very very soon.
So
* is it required that jars are signed for an incubation project? The EDP doesn't mention anything about that. IIRC, signing is a SimRel requirement. (I've sent a mail to architecture council to clarify that)
* Can a 1st release happen manually to unlock us? And then automation can be taken care of for the next release?
* Releasing to repo.eclipse.org would be enough as a 1st step for us. Going to Central isn't a requirement for extenders/adopters.

As we're really in need of this release, is there anything I could try to help with so we can adopt the new artifacts and package name ASAP in Wild Web Developer?
(I did chat with @angelozerr , and apparently the target is in 2 weeks, it's be much better for us if it could be 2 days ;)

**Comment by mickaelistria** [comment]
> s it required that jars are signed for an incubation project? The EDP doesn't mention anything about that. IIRC, signing is a SimRel requirement. (I've sent a mail to architecture council to clarify that)

The Architecture Council confirmed that signing isn't mandatory for a release.

**Comment by fbricon** [comment]
The targeted milestone is feature complete, so we can move forward with an earlier release. Will try to complete it this week.

**Comment by fbricon** [comment]
0.11.0 is out. kinda. You can use
```xml
<dependency>
    <groupId>org.lemminx</groupId>
    <artifactId>org.eclipse.lemminx</artifactId>
    <version>0.11.0</version>
    <!-- classifier:uber includes all dependencies -->
    <classifier>uber</classifier>
</dependency>
```
You will have to reference the Maven repository hosting the dependency you need. E.g. for Maven, add this repository to your pom.xml or settings.xml :
```xml
<repository>
  <id>lemminx-releases</id>
  <url>https://repo.eclipse.org/content/repositories/lemminx-releases/</url>
  <snapshots>
    <enabled>false</enabled>
  </snapshots>
  <releases>
    <enabled>true</enabled>
  </releases>
</repository>
```

Main issues for now are:
- I failed to copy the released bits on download.eclipse.org in the release script, will need to fix it manually, but it's no big deal since the jars are already in nexus
- Only the core org.eclipse.lemminx-0.11.0.jar is signed. but not when included in the zipped p2 repo. Uber jar is not signed.
- Maven Central release not done.

**Comment by mickaelistria** [comment]
Thanks a lot!

**Comment by fbricon** [comment]
I realized the groupid was borked, so I re-released 0.11.0 as 
```
<dependency>
    <groupId>org.eclipse.lemminx</groupId>
    <artifactId>org.eclipse.lemminx</artifactId>
    <version>0.11.0</version>
    <!-- classifier:uber includes all dependencies -->
    <classifier>uber</classifier>
</dependency>
```
